app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :update_comments_counter
+
+  def update_comments_counter
+    post.comments_counter = post.comments.count
```

Comment:
- [ ] [OPTIONAL] Instead of using `count` for the counter, you can use `increment` Ruby method like this: `post.increment!(:comments_counter)`.